build.gradle

```diff
@@ -240,6 +240,12 @@ task osInfo {
     }
 }
 
+if (JavaVersion.current().isJava8Compatible()) {
+    tasks.withType(Javadoc) {
+        options.addStringOption('Xdoclint:none', '-quiet')
+    }
+}
+
```

Comment:
Please remove these additions, as we do not want to invest in javadoc at this moment.